javascript/ql/lib/semmle/javascript/security/dataflow/ZipSlipQuery.qll

```diff
@@ -11,11 +11,11 @@ import javascript
 import ZipSlipCustomizations::ZipSlip
 
 // Materialize flow labels
-private class ConcretePosixPath extends TaintedPath::Label::PosixPath {
+private class ConcretePosixPath extends TaintedPath::Path::PathLabel {
```

Comment:
When I saw this I thought you were materializing the labels for all platforms here.  
You are, but the only platform that is activated by this file is the Posix platform.  

I think you could add in the comment of these "label-materializations" that only the Posix platform exists for this query, and add in the QLDoc for `abstract class Platform extends string` that only the Posix platform exists by default.